LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/LoRaPhysical/Rxpk.cs

```diff
@@ -122,5 +122,45 @@ public uint GetModulationMargin()
 
             return (uint)signedMargin;
         }
+
+        public (double freq, string datr) GetDownstreamRX2DRAndFreq(string devEUI, string nwkSrvRx2Dr, double nwkSrvRx2Freq, Region loraRegion, int? rx2DrFromTwins)
+        {
+            double freq = 0;
+            string datr = string.Empty;

```

Comment:
init to null? looks like you always going to assign a string?

Reply:
Well, normally StyleCop complains when I don't initalize variables, So I guess I just initialize everything now. I could change this to remove the String.Empty